### ISSUE TITLE
Support running Sdk and devcluster in a container on windows

### ISSUE THREAD

**Issue by aL3891** [Microsoft/service-fabric#42]
It would be very helpful for devops scenarios to 1) be able to publish an app to a remote cluster from inside a container and 2) run a dev cluster inside a container for testing purposes. 

As I understand it, the sdk/runtime is able to run in a container on other platforms than windows, but I have not been able to get it to install on a windows container.

I understand this is also a work in progress but it would be cool if the progress/things remaining could be tracked here on GitHub as well, so what are currently the things that are missing for this to work?

**Comment by samedder** [comment]
We'll look into if this is feasible and work on it in the future. We're aware it doesn't work, but we think this is a valid scenario.

**Comment by Flern** [comment]
Note that there is an issue in the old repo that others are tracking with a similar use case:
[Error installing Service Fabric Runtime in windowsservercore:1709 docker container](https://github.com/Azure/service-fabric-issues/issues/711)

This specifically references the runtime that is required by the SDK.  I assume the same problems (kernel driver dependency on Windows) drives the lack of local clusters in containers as well.

This feature would allow a real CI pipeline to be utilized without driving everyone using Service Fabric to Linux-based clusters since this behavior is natively supported on Linux.

**Comment by samedder** [comment]
Yep, it's something we're pushing for however this isn't a simple change so we'll update this issue once we understand how it will be prioritized in future releases.

**Comment by Mortana89** [comment]
So is this still on the radar?
We'd like to use this to run E2E tests as well within the container
